pruner_config.toml

```diff
@@ -0,0 +1,7 @@
+[pruner.folder1]
```

Comment:
isn't a dictionary better data structure to use here for storing config using .yaml file.. something like -
```
prune-directory:
    folder-1-path: day_count_1
    folder-2-path: day_count_2
```

Reply:
toml is like a json file but better. toml is very similar to yaml 
[pruner.harshith]
path = "harshith"
days_to_live = 10

[pruner.perf-ci]
path = "perf-ci"
days_to_live = 60
an example of a pruner_config.toml file

Reply:
yes have changed the way the toml looks : 

pruner = [
 { path = "folder1 name", days_to_live =  },
 { path = "folder2 name", days_to_live =  },
 ....
]